src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -58,6 +58,7 @@
         "symfony/string": "^5.4|^6.0",
         "symfony/translation": "^5.4|^6.0",
         "symfony/twig-bundle": "^5.4|^6.0",
+        "symfony/uid": "^6.1",
```

Comment:
`^5.4|^6.0` and we throw if the argument_value_resolver is used?

Reply:
I mean we can remove the conflict.

Reply:
let's allow 5.4